src/molecules/BarSm.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Composition } from 'atomic-layout'
+import { IoIosSearch, IoMdPerson, IoMdCart } from 'react-icons/io'
+import styled from 'styled-components'
+
+const Container = styled.div`
+  background-color: #262626;
+  justify-content: space-around;
+`
+
+const templateSm = `
+	myAccount search card
+`
+
+export const BarSm: React.FC = () => {
+  return (
+    <Composition template={templateSm} as={Container} padding="10px">
+      {Areas => (
+        <>
+          <Areas.MyAccount>
+            <a href="#">
+              <IoIosSearch size="35" color="#f3f3f3" />
```

Comment:
It looks like you use `BarSm` to give a different icon size, isn't it?

If that's the case, you can make the icon component responsive using `useResponsiveComponent` hook from `atomic-layout`:

```js
import { useResponsiveComponent } from 'atomic-layout'

const SearchIcon = useResponsiveComponent(IoIosSearch)

const BarSm = () => {
  return <SearchIcon size="22" sizeMd="35" color="#f3f3f3" />
}
```

Wrapping any React component in `useResponsiveComponent` makes all props of that component responsive, so they support suffixes to assign different values. 

- [`useResponsiveComponent` documentation](https://redd.gitbook.io/atomic-layout/api/hooks/use-responsive-component)